packages/react-instantsearch/src/components/__snapshots__/Breadcrumb.test.js.snap

```diff
@@ -0,0 +1,223 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`Breadcrumb has a separator prop that can be a custom component 1`] = `
+<div
+  className="ais-Breadcrumb__root"
+>
+  <div
```

Comment:
I think that a breadcrumb item should be a span, so that the default rendering without css is like what we expect: 

div|span
---|---
<img width="171" alt="screen shot 2017-08-03 at 15 45 06" src="https://user-images.githubusercontent.com/6270048/28924660-c8a7a9ea-7862-11e7-9c46-e8567315ac3f.png"> | <img width="328" alt="screen shot 2017-08-03 at 15 45 00" src="https://user-images.githubusercontent.com/6270048/28924661-c8bd5236-7862-11e7-85cf-0a71cdcbf0ff.png">